packages/react-app/src/services/bounty.service.ts

```diff
@@ -0,0 +1,154 @@
+import Bounty, { BountyBoardProps } from '../models/Bounty';
```

Comment:
This was previously in the bounty controller for getBountes, now in a separate services file and folder